src/index.js

```diff
@@ -8,7 +8,7 @@ import { buildMockLocalContext, mockApi } from '@graasp/apps-query-client';
 import App from './components/App';
 import { ENABLE_MOCK_API, SENTRY_DSN } from './config/constants';
 import { SENTRY_ENVIRONMENT, SENTRY_TRACE_SAMPLE_RATE } from './config/sentry';
-import buildDatabase from './data/db';
+import buildDatabase2 from './data/db2';
```

Comment:
Why db2?

Reply:
Because to better see how the `Analytics` and the `result tables` would resemble with a little bit more data, I created this new `db2`, that contains 10 users, and 10 questions. As before with only 4 questions and 3 users, I found that it was a little poor to have a good representation.

I hesitated to remove the initial `db`, but I thought it might still be useful someday, if we wanted to display less data during development.

Do you think I should put back `db` ?

Reply:
I'm okay with having another db, but maybe have a better name for it.